### PR TITLE
Edit status 2

### PR DESCRIPTION
**Pull request by kimsunhammond**
Enum is showing and editable from the edit task page. The database also updates with the edits to the task status.

### CODE REVIEW

File: src/main/java/org/launchcode/liftoffgroupproject/Test Data.sql
Comment (allylb-code) [inline src/main/java/org/launchcode/liftoffgroupproject/Test Data.sql:1]: Everything seems to be working on MySQL but after I dropped my tables it did not re-insert these values into MySQL. Are these tasks something we want to keep or are we having it blank so the user can make their own tasks?

File: src/main/resources/templates/list.html [hunk 1]
```diff
@@ -25,9 +26,20 @@
               <td th:text="${task.description}">description</td>
               <td th:text="${task.startDate}" >07/10/23</td>
               <td th:text="${task.dueDate}">07/14/23</td>
+              <td th:text="${task.taskProgressEnum.getDisplayStatus()}">progress</td>
               <td>
+<!--                  <a th:href="@{/edit/{id}(id=${task.id})}">Edit</a>-->
```
Comment (allylb-code) [inline src/main/resources/templates/list.html]: Delete commented out code?

File: src/main/resources/templates/taskDetails/taskDetails.html [hunk 2]
```diff
@@ -1,20 +1,51 @@
...
+            </li>
+
+            <li th:case="${T(org.launchcode.liftoffgroupproject.models.TaskProgressEnum).COMPLETED}" class="bg-success list-group-item">
+                <span>Status: </span>
+                <span th:text="${task.taskProgressEnum.getDisplayStatus()}">Completed</span>
+            </li>
+
+        </th:block>
+
+<!--        <span>Status: </span>-->
```
Comment (allylb-code) [inline src/main/resources/templates/taskDetails/taskDetails.html]: Delete commented out code?

**Review by allylb-code — APPROVED** [review]
Great work! Code look good and it's working perfectly. :) Just a few comments from me.

File: src/main/java/org/launchcode/liftoffgroupproject/Test Data.sql
Comment (kimsunhammond) [reply src/main/java/org/launchcode/liftoffgroupproject/Test Data.sql:1]: I have been keeping it blank so the user can make their own tasks. If we want to use the Test Data, I believe we will need to reformat it and then copy and paste it into a table in MySQL, like we did at the very beginning.

**Pull request merged**